packages/components/src/components/tabs/_tabs.scss

```diff
@@ -710,8 +710,13 @@
       @include focus-outline('reset');
       @include type-style('body-short-01');
 
+      @if feature-flag-enabled('enable-v11-release') {
+        width: auto;
```

Comment:
Do we need to explicitly set it to `auto` because of another `width` declaration? I had assumed this was the default value so just wanted to check 👀

Reply:
You're right, `auto` isn't needed. I'll remove 👍